pkg/provider/aws/dryrun.go

```diff
@@ -51,32 +51,12 @@ func (a *Client) checkInstanceTypes() error {
 }
 
 func (a *Client) checkImages() error {
-	var nextToken *string
-
-	for {
-		// Use the DescribeImages API to get a list of supported images in the current region
-		resp, err := a.ec2.DescribeImages(context.TODO(), &ec2.DescribeImagesInput{
-			NextToken: nextToken,
-		},
-		)
-		if err != nil {
-			return err
-		}
-
-		for _, image := range resp.Images {
-			if *image.ImageId == *a.Spec.Instance.Image.ImageId {
-				return nil
-			}
-		}
-
-		if resp.NextToken != nil {
-			nextToken = resp.NextToken
-		} else {
-			break
-		}
+	// Check if the given image is supported in the region
```

Comment:
Question: Why is this in the `dryrun` file?

Reply:
Good catch, will move